tests/Base.php

```diff
@@ -213,6 +213,15 @@ public function testRequestMethod(): void
         $response = $this->execute(method: 'PATCH', headers: ['x-action' => 'requestMethod']);
         self::assertEquals(200, $response['code']);
         self::assertEquals('PATCH', $response['body']);
+
+        $response = $this->execute(method: 'HEAD', headers: ['x-action' => 'requestMethod']);
+        self::assertEquals(200, $response['code']);
+        // Some runtimes either return '', while some return 'HEAD'
+        // self::assertEquals('', $response['body']);
```

Comment:
Let's enable this test and allow both. Refference to allowing 2 options in a test: https://github.com/ItzNotABug/open-runtimes/blob/684549e12f49def843cc2162428ab2f7c5588f33/tests/Base.php#L241

Reply:
Makes sense. I'll enable this one.